data/imports/fdx.tmpl

```diff
@@ -58,16 +58,18 @@ clients:
   scopes:
   - manage_openbanking_consents
   token_endpoint_auth_method: client_secret_basic
-# developer tpp
+
+# financroo / developer tpp
 - tenant_id: {{ .tenant_id }}
   authorization_server_id: {{ .server_id }}
   client_id: bugkgm23g9kregtu051g
-  client_name: Developer TPP
+  client_name: Financroo
```

Comment:
General remark. So far, Developer TPP / Financroo was one app. We would like to split it (there should be 2 separate oauth clients).
This can be done as a separate PR.

Reply:
will do